app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryAttachments.js

```diff
@@ -1,108 +1,99 @@
-      <ExpandedStorySection
-        title={I18n.t('story.attachments')}
-        identifier="attachments"
-      >
-        {
-          !disabled && (
-            <Dropzone
-              onDrop={this.onFileDrop}
-              accept={acceptedMimeTypes()}
-              disabled={this.state.loading}
-              multiple={false}
-              data-id="dropzone"
-            >
-              {({ getRootProps, getInputProps, isDragActive, isDragReject }) => {
-                const className = this.dropzoneClassName({ isDragActive, isDragReject })
+  return (
+    <ExpandedStorySection
+      title={I18n.t('story.attachments')}
+      identifier="attachments"
+    >
+      {
+        !disabled && (
+          <Dropzone
+            onDrop={onFileDrop}
+            accept={acceptedMimeTypes()}
+            disabled={loading}
+            multiple={false}
+            data-id="dropzone"
+          >
+            {({ getRootProps, getInputProps, isDragActive, isDragReject }) => {
+              const className = classname(
+                'btn btn-info attachments-dropzone',
+                {
+                  'attachments-dropzone--loading': loading,
+                  'attachments-dropzone--reject': isDragReject,
+                  'attachments-dropzone--drag': isDragActive
+                }
+              );
 
-                return (
-                  <div
-                    {...getRootProps()}
-                    className={`btn btn-success attachments-dropzone ${className}`}
-                  >
-                    <input {...getInputProps()} />
-                    <i className="mi md-20">cloud_upload</i>
-                    <div>
-                      {!isDragReject ?
-                        I18n.t('upload_new_file') :
-                        I18n.t('reject_new_file')}
-                    </div>
+              return (
+                <div
+                  {...getRootProps()}
+                  className={className}
+                >
+                  <input {...getInputProps()} />
+                  <i className="mi md-20">cloud_upload</i>
+                  <div>
+                    {!isDragReject ?
+                      I18n.t('upload_new_file') :
+                      I18n.t('reject_new_file')}
                   </div>
-                )
-              }}
-            </Dropzone>
-          )
-        }
-        <AttachmentsList
-          files={story._editing.documents}
-          onDelete={onDelete}
-          disabled={disabled}
-        />
-      </ExpandedStorySection>
-    );
-  }
+                </div>
+              )
+            }}
+          </Dropzone>
+        )
+      }
+      {
+        needsToSave && <ExpandedStoryAttachmentsInfo data-id="attachments-info" />
+      }
+      <AttachmentsList
+        files={story._editing.documents}
+        onDelete={onDelete}
+        disabled={disabled}
+      />
+    </ExpandedStorySection>
+  );
 }
 
 ExpandedStoryAttachments.propTypes = {
```

Comment:
Missing prop type `needsToSave`. 
I believe you set `defaultProps` as `false` and then, update this line of code:
```diff
- needsToSave={story.needsToSave || false}
+ needsToSave={story.needsToSave}
```

Reply:
Nice, I will update it